frontend/src/metabase/lib/formatting/numbers.tsx

```diff
@@ -83,7 +83,12 @@ export function formatNumber(
   } else {
     try {
       let nf;
-      if (number < 1 && number > -1 && options.decimals == null) {
+      if (
+        number < 1 &&
+        number > -1 &&
+        options.decimals == null &&
+        options.number_style !== "percent"
```

Comment:
Based on the comment below, this block handles formatting small numbers i.e. those between -1 and 1, and has decimal points.

This causes `percent` to fail. Since `0.678` will be rounded to `68%` as opposed to `67.8%`

so, it should be safe to ignore any `percent` in this block.